chrome_manifest_v3/class_app.js

```
@@ -0,0 +1,848 @@
+  /**
+   * Correctly escape RegExp
+   */
+  escapeRegExp(str) {
+    return str.replace(/([.*+?^=!:${}()|\[\]\/\\])/g, '\\$1');
+  }
+
+  /**
+   * Utility routine to replace variables
+   */
+  replacer(text, dict) {
+    let self = this;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'self' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)